PocketKit/Sources/PocketKit/Collections/CollectionViewController.swift

```diff
@@ -84,6 +84,7 @@ class CollectionViewController: UIViewController {
         collectionView.delegate = self
 
         collectionView.register(cellClass: LoadingCell.self)
```

Comment:
should we remove the loading cell implementation / code now that it is being replaced with the empty cell?